tsl/test/expected/dist_query.out

```diff
@@ -2444,16 +2444,18 @@ GROUP BY 1,2
 HAVING device > 4
 ORDER BY 1,2
 
-                                                                                                                                                                                                                   QUERY PLAN                                                                                                                                                                                                                    
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- Custom Scan (DataNodeScan)
-   Output: (time_bucket('@ 2 days'::interval, hyper."time")), hyper.device, (avg(hyper.temp))
-   Relations: Aggregate on (public.hyper)
-   Data node: data_node_1
-   Fetcher Type: Row by row
-   Chunks: _dist_hyper_1_1_chunk
-   Remote SQL: SELECT public.time_bucket('@ 2 days'::interval, "time"), device, avg(temp) FROM public.hyper WHERE _timescaledb_internal.chunks_in(public.hyper.*, ARRAY[1]) AND (("time" >= '2019-01-01 00:00:00-08'::timestamp with time zone)) AND (("time" <= '2019-01-01 15:00:00-08'::timestamp with time zone)) AND ((device > 4)) AND ((device = 1)) GROUP BY 1, 2 ORDER BY public.time_bucket('2 days'::interval, "time") ASC NULLS LAST
-(7 rows)
+                                           QUERY PLAN                                            
+-------------------------------------------------------------------------------------------------
```

Comment:
Interesting change. So the output is basically NULL?

Reply:
Yes, the table is excluded by the constraint exclusion because the conditions in WHERE and HAVING are contradictory, so we get the empty Result node. Having GroupAggregate + Sort on top of Result looks somewhat weird, but not incorrect.